javascript/src/main/pages/TutorAssignment/TutorAssignments.js

```diff
@@ -39,52 +95,81 @@ const TutorAssignment = () => {
     getToken, mutateCourses, onDeleteTutorError
```

Comment:
Instead of `mutateCourse` we should replace this with `mutateTutorAssignments`, this should fix the bug observed where there is a noticeable delay between deleting a tutor assignment, and the results being updated on the table.

Reply:
Fixed bug, it did help to remove the delay